BUILD.bazel

```diff
@@ -55,39 +64,68 @@ js_library(
     visibility = ["//visibility:public"],
 )
 
+npm_package(
+    name = "npm_package",
+    srcs = [
+        "AUTHORS",
+        "BUILD.bazel",
+        "CODE_OF_CONDUCT.md",
+        "CONTRIBUTING.md",
+        "CONTRIBUTORS",
+        "DEVELOPING.md",
+        "LICENSE",
+        "README.md",
+        "WORKSPACE",
+        "defs.bzl",
+        "package.bzl",
+        "package.json",
+        "version.bzl",
+        "//devserver:npm_package_assets",
+        "//internal:npm_package_assets",
+        "//third_party/github.com/bazelbuild/bazel/src/main/protobuf:npm_package_assets",
+        "//ts_auto_deps:npm_package_assets",
+    ],
+    replacements = {
+        # Do a simple replacement needed to make the local development differ
+        # from how our release is used.
+        # TODO(mgechev): add devserver replacement back in once compiled binaries work.
+        #                current broken on OSX at least with error:
+        #                `devserver-darwin_amd64: cannot execute binary file`
+        # "//devserver:devserver": "//devserver:server",
+        "0.0.0-COMPAT_VERSION": COMPAT_VERSION,
+    },
+    deps = [
+        # TODO(mgechev): add devserver binaries back in once they work (see above)
+        # "//devserver:devserver-darwin",
+        # "//devserver:devserver-linux",
+        # "//devserver:devserver-windows",
+        "//internal:tsc_wrapped",
+    ],
+)
+
+# Produces the release we publish to GitHub. Users download this starlark package
```

Comment:
are we still going to publish a release to github? If you're not using the `@bazel/typescript` package I'd rather you just fetch our whole repo (and build things from source)

Reply:
Ideally we can archive up the generated npm package and publish two archives: one for rules_typescript and one for rules_karma. The trouble is the `pkg_tar` rules adds the `npm_package` folder to the archive.

Alternately, users can http_archive the repo for rules_typescript and http_archive the repo with a strip-prefix of `internal/karma` for the rules_karma workspace.